src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -107,6 +107,9 @@ The .NET Foundation licenses this file to you under the MIT license.
     <!-- Exclude System.IO.Compression.Native.dll -->
     <FrameworkAssemblies Include="$(IlcPath)\framework\*.dll" Exclude="$(IlcPath)\framework\*.Native.dll" />
   </ItemGroup>
+  <ItemGroup Condition="'$(IlcPgoOptimize)' == 'true'">
+    <MibcFile Include="$(IlcPath)\mibc\*.mibc" />
```

Comment:
Do we plan to ship framework *.mibc files for crossgen2 as well? Or is crossgen2 going to read the mibc data attached to .dlls? I am wondering whether this should be on the same plan.

Reply:
Crossgen2 embeds this data into ReadyToRun formats - we don't currently ship with the readytorun compiled framework because it's just a bunch of useless junk.

Yes, eventually, we'll probably want to parse this out of the R2R headers.